src/components/RedLiner/helpers/components.ts

```diff
@@ -0,0 +1,15 @@
+import { ComponentType } from 'react';
+import InfoBox, { IInfoBoxProps } from '../../InfoBox/InfoBox';
+
+export type ComponentsConfig = {
+  InfoBox: ComponentType<IInfoBoxProps>;
```

Comment:
Only `InfoBox` is customizable now. Planning on adding support for `Line` after `InfoBox`.